### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mhairimca**
Added expiring multiparty contracts

### CODE REVIEW

File: docs/getting-started/welcome.md [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
-
-You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md).
-
-## For UMA Tokenholders
-
-If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
-
-## Contact 
-If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
+---
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:1]: Same comment as on previous PRs - don't want it to overwrite our changes.

**Review by Tommy1231232 — COMMENTED** [review]
Nice job!!

File: docs/getting-started/welcome.md (hunk 1 above)
Comment (mhairimca) [reply docs/getting-started/welcome.md:1]: Welcome file is now overwritten by PR143

**Comment by smb2796** [comment]
Closing. Will be replaced with updated pr.

**Pull request closed without merging**